### PR TITLE
PT-2046

### PR DESCRIPTION
**Pull request by tmarathe**

### CODE REVIEW

File: components/xwiki-platform-users/xwiki-platform-users-api/pom.xml [hunk 1]
```diff
@@ -63,6 +63,11 @@
       <version>${xwiki.version}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.commons</groupId>
+      <artifactId>xwiki-commons-component-api</artifactId>
+      <version>7.0.1</version>
```
Comment (sdumitriu) [inline components/xwiki-platform-users/xwiki-platform-users-api/pom.xml]: Don't hardcode versions like this, use `${xwiki.version}`.

This makes it easier to upgrade the XWiki version, since it only requires changing one parameter in one place, instead of searching for all the individual dependencies in all the pom files.

File: components/patient-data/impl/pom.xml [hunk 2]
```diff
@@ -122,6 +122,12 @@
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
     </dependency>
+    <dependency>
```
Comment (sdumitriu) [inline components/patient-data/impl/pom.xml]: This isn't needed.

File: components/obo2solr/pom.xml [hunk 3]
```diff
@@ -46,6 +46,11 @@
       <groupId>org.slf4j</groupId>
       <artifactId>slf4j-api</artifactId>
     </dependency>
+    <dependency>
```
Comment (sdumitriu) [inline components/obo2solr/pom.xml]: This isn't needed.

Comment (tmarathe) [inline components/obo2solr/pom.xml]: Yup, I just noticed unnecessary changes to pom files, removing them now.
